asv_bench/benchmarks/index_object.py

```diff
@@ -161,9 +161,7 @@ def setup(self, dtype):
         self.sorted = self.idx.sort_values()
         half = N // 2
         self.non_unique = self.idx[:half].append(self.idx[:half])
-        self.non_unique_sorted = (
-            self.sorted[:half].append(self.sorted[:half]).sort_values()
-        )
+        self.non_unique_sorted = self.sorted[:half].repeat(2)
```

Comment:
Is the sort_values here no longer required?

Reply:
Thats correct, it is no longer required. The array is already sorted so using `repeat` here will maintain the sort.